README.md

```diff
@@ -1,2 +1,38 @@
-# idea-hub
-A hub to share ideas and innovations
+# idea-hub-new
+
+## Project setup
+```
+npm install
+```
+
+### Compiles and hot-reloads for development
+```
+npm run serve
+```
+
+### Compiles and minifies for production
+```
+npm run build
+```
+
+### Run your unit tests
+```
+npm run test:unit
+```
+
+### Run your end-to-end tests
+```
+npm run test:e2e
+```
+
+### Lints and fixes files
+```
+npm run lint
+```
+
+### Customize configuration
+See [Configuration Reference](https://cli.vuejs.org/config/).
+
+## Contributors:
+
```

Comment:
As a skeleton this is fine. Please do update the documentation regularly on next levels